src/components/BookCard.js

```diff
@@ -0,0 +1,52 @@
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+
+/* eslint-disable react/prop-types */
+const BookCard = ({ books }) => {
```

Comment:
Please remove the block comment to enable Eslint, let's not disable ESLint in any of our files, they represent style guidelines that you share with your team - which is a group of all Microverse students. 💪

- [ ] PropTypes serves as the method you can use to ensure the correct datatype is passed for each prop, meaning whatever is rendered is correct. 

- [ ] It can also be used to ensure a value is passed, set default values, and lots more 🤗, please add them 🛠

Reply:
- Added `PropTypes` for data validation purposes.